test/kuttl/tests/cinder_scale/02-scale-up-cinder-api.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      oc patch cinder -n $NAMESPACE  cinder --type='json' -p=\
+       '[{"op": "replace", "path": "/spec/cinderAPI/replicas", "value":1}]'
```

Comment:
Don't you want to set the value to 3?

Reply:
Ack, the test passed with 3 replicas in my local env. Updated the patch per local tests